include/boost/url/params.hpp

```diff
@@ -160,37 +134,34 @@ class params
     */
     using difference_type = std::ptrdiff_t;
 
-    /** <!-- @name Members -->
-        <!-- @{ -->
-     */
-
-    /** Assignment
+    /** View assignment
 
         After the assignment, both views will point to
         the same underlying object.
+
     */
     params&
     operator=(params const&) & = default;
 
-    /** Assignment from initializer list
-
-        Assigns @ref params from a list of @ref params::value_type.
+    /** Assignment
 
-        Each instance of @ref params::value_type is a view of a query <key, value> pair.
+        Assignment from initializer list.
```

Comment:
Well this is not a sentence, since there is no verb. How about "The query parameters are replaced by the contents of the initializer list. The behavior is undefined if any strings in the list reference part or all of the URL."

Reply:
Is this really supposed to be public though? Only `assign(FwdIt first, FwdIt last)` calls it.

Reply:
OK. I was trying to reduce things in this PR but I think I went too far. :)